pages/proposal/[id].js

```diff
@@ -265,25 +268,24 @@ const Proposal = ({ id, defaultProposalData }) => {
     }
   };
 
-  useEffect(fetchProposal, [proposals]);
-  useEffect(fetchReceipt, [authed]);
+  useEffect(refreshVotesAndState, [tick]);
```

Comment:
if you need tick and provider. might be a good idea to add a provider to the dependency array as well